website/views/static/faq.jade

```diff
@@ -0,0 +1,115 @@
+extends ./layout
+
+block vars
+  - var layoutEnv = env
+
+block title
+  title Habitica &VerticalLine;&nbsp;
+    =env.t('FAQ')
+
+block content
+  script(type='text/javascript').
```

Comment:
Snippet cobbled together from several StackOverflow answers and tweaked to suit. (It's jQuery largely because Bootstrap's accordion plugin is.) Better approaches? Worth moving out to its own .js file?